flyway/sql/V1.0.0__init.sql

```diff
@@ -1,13 +1,55 @@
 -- テーブル作成
--- ユーザマスタ
-CREATE TABLE m_user
-(
-    id                        int unsigned NOT NULL AUTO_INCREMENT COMMENT 'ID',
-    created_at                datetime            DEFAULT CURRENT_TIMESTAMP COMMENT '登録日時',
-    updated_at                datetime            DEFAULT CURRENT_TIMESTAMP ON UPDATE CURRENT_TIMESTAMP COMMENT '更新日時',
-    deleted_at                datetime            DEFAULT NULL COMMENT '削除日時',
-    PRIMARY KEY (id)
-) ENGINE = InnoDB
-  DEFAULT CHARSET = utf8mb4
-  COLLATE = utf8mb4_unicode_ci comment ='ユーザマスタ'
-;
\ No newline at end of file
+
+-- ユーザーデータテーブル
+CREATE TABLE users(
+    user_id                     int(16) NOT NULL AUTO_INCREMENT,
+    login_id                    varchar(32) NOT NULL UNIQUE,
+    display_name                varchar(32),
+    email                       varchar(64) NOT NULL,
+    password                    varchar(42) NOT NULL,
+    created_at                  datetime NOT NULL DEFAULT CURRENT_TIMESTAMP,
+    PRIMARY KEY (user_id)
+)ENGINE = InnoDB
+   DEFAULT CHARSET = utf8mb4
+   COLLATE = utf8mb4_unicode_ci
+ ;
+
+
+-- 投稿データテーブル
+CREATE TABLE posts(
+    post_id                     int(32) NOT NULL,
+    user_id                     int(16) NOT NULL,
+    text                        varchar(140) NOT NULL,
+    created_at                  datetime NOT NULL DEFAULT CURRENT_TIMESTAMP,
+
+    FOREIGN KEY(user_id) REFERENCES users(user_id),
+    PRIMARY KEY(post_id)
+
+)ENGINE = InnoDB
+   DEFAULT CHARSET = utf8mb4
+   COLLATE = utf8mb4_unicode_ci
+ ;
+
+
+-- いいねデータテーブル
+CREATE TABLE likes(
+    user_id                     int(16) NOT NULL,
+    post_id                     int(32) NOT NULL,
+    created_at                  datetime NOT NULL DEFAULT CURRENT_TIMESTAMP,
+
+    FOREIGN KEY(user_id) REFERENCES users(user_id),
+    FOREIGN KEY(post_id) REFERENCES posts(post_id),
+    PRIMARY KEY(user_id, post_id)
+)ENGINE = InnoDB
+   DEFAULT CHARSET = utf8mb4
+   COLLATE = utf8mb4_unicode_ci
+ ;
+
+insert into users values(1,"taro","", "taro@mail.com", "fh83y4hnuthija89fdsa", "2022-08-19"),
```

Comment:
[imo]
* 予約語は大文字だと読みやすいです
* カンマの後の半角スペースが入っていたり入っていなかったりするのが気になります

```suggestion
INSERT INTO users VALUES(1, "taro", "", "taro@mail.com", "fh83y4hnuthija89fdsa", "2022-08-19"),
```

SQLの書き方を統一するコストをかけるより、フォーマッタにおまかせすると良いと思います。

Reply:
修正しました！ご確認よろしくお願いします🙇
フォーマッタにかけるというのはこのような感じで良いでしょうか？
https://github.com/deliciouspasta/test-project/pull/1/commits/15983e5e3f88ae99e91004ed27c2a37a0ba555e2

Reply:
@deliciouspasta 
すみません、こちらのコメント見逃していて別にコメントしてしまいました。

下記コメントご確認お願いします。
https://github.com/deliciouspasta/test-project/pull/1#discussion_r951195665